WMF Framework/WMFLocalization.m

```diff
@@ -0,0 +1,20 @@
+#import "WMFLocalization.h"
+
+static NSString *const WMFLocalizationBundleIdentifier = @"org.wikimedia.WMF";
+
+NSString *WMFLocalizedStringWithDefaultValue(NSString *key, NSURL *url, NSBundle *bundle, NSString *value, NSString *comment) {
```

Comment:
For non site-specific localizations, I'd suggest we stick to `NSLocalizedString` or `NSLocalizedStringWithDefaultValue` but I'd be open to switching everything to a `WMF` method if that'd be easier and more consistent.